server/src/main/scala/drt/server/feeds/bhx/BHXArrivals.scala

```diff
@@ -0,0 +1,87 @@
+package drt.server.feeds.bhx
+
+import drt.shared.Arrival
+import javax.xml.datatype.XMLGregorianCalendar
+import org.joda.time.DateTime
+import services.SDate
+import uk.co.bhx.online.flightinformation.{FlightRecord, ScheduledFlightRecord}
+
+sealed trait BHXArrivals {
+
+  def convertToUTC(feedDate: XMLGregorianCalendar): Option[String] = {
+    val date = feedDate.toGregorianCalendar.getTime
+
+    date.getTime match {
+      case 0L => None
+      case _ =>
+        val datetime = new DateTime(date.getTime)
+
+        import org.joda.time.DateTimeZone
+        import org.joda.time.format.ISODateTimeFormat
+
+        Some(datetime.withZone(DateTimeZone.UTC).toString(ISODateTimeFormat.dateTime))
+    }
+  }
+
+}
+
+trait BHXLiveArrivals extends BHXArrivals {
+
+  def toLiveArrival(flightRecord: FlightRecord): Arrival =
+    new Arrival(Operator = "",
+      Status = flightRecord.getFlightStatus match {
+        case "Arrived" => "A"
```

Comment:
Let's leave the statuses as they are from the feed. They look very readable to me. Flight stats changes them to single letters for some reason which then the PCP app converts to something slightly more meaningful but we can just pass these through